operations/mimir/common.libsonnet

```diff
@@ -71,4 +71,9 @@
       topologySpreadConstraints.withWhenUnsatisfiable('ScheduleAnyway') +
       topologySpreadConstraints.withMaxSkew(maxSkew),
     ),
+
+  mimirVolumeMounts()::
```

Comment:
I'm wondering if we should use `mimirVolumeMounts` or `mimirVolumeMounts()` here. WDYT @pracucci ?

Reply:
A field is overrideable, the function is not. Probably the field is more flexible for this use case.